.github/workflows/lint.yml

```diff
@@ -0,0 +1,27 @@
+name: Lint
+
+on: [push, pull_request]
```

Comment:
If we use an appraoch like here: https://github.com/OpenTOSCA/winery/blob/ustutt/.github/workflows/build.yml#L4
we avoid duplicate runs of the same workflow.